src/main/resources/db/migration/V1_72__appointment_referral_data.sql

```diff
@@ -0,0 +1,22 @@
+Update appointment apo
+set referral_id = (
```

Comment:
_not a blocker_
This kind of thing feels to be simpler through one-off management tasks, does spring have a capacity to run those?

I'm thinking of things like in Rails/Phoenix where we could `kubectl exec` into a pod and run `rake/mix task` as a one-off